ci/test/cargo-test/mzcompose.py

```diff
@@ -47,25 +49,91 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         f"postgres://postgres:postgres@localhost:{c.default_port('postgres')}"
     )
     cockroach_url = f"postgres://root@localhost:{c.default_port('cockroach')}"
-    spawn.runv(
+
+    env = dict(
+        os.environ,
+        ZOOKEEPER_ADDR=f"localhost:{c.default_port('zookeeper')}",
+        KAFKA_ADDRS="localhost:30123",
+        SCHEMA_REGISTRY_URL=f"http://localhost:{c.default_port('schema-registry')}",
+        POSTGRES_URL=postgres_url,
+        COCKROACH_URL=cockroach_url,
+        MZ_SOFT_ASSERTIONS="1",
+        MZ_PERSIST_EXTERNAL_STORAGE_TEST_S3_BUCKET="mtlz-test-persist-1d-lifecycle-delete",
+        MZ_PERSIST_EXTERNAL_STORAGE_TEST_POSTGRES_URL=cockroach_url,
+    )
+
+    coverage = bool(os.getenv("CI_COVERAGE_ENABLED", False))
+
+    if coverage:
+        (ROOT / "coverage").mkdir(exist_ok=True)
+        # Currently not required, since cargo-llvm-cov does this automatically
+        # env["LLVM_PROFILE_FILE"] = f"{ROOT}/coverage/cargotest-%p-%9m%c.profraw"
+        # env["RUSTFLAGS"] = " ".join(rustc_flags.coverage)
+
+        # cargo-llvm-cov wants to use rustup to get its runtime dependencies
+        subprocess.run(
+            "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y",
+            shell=True,
+        )
+        subprocess.run(
+            ["cargo", "install", "cargo-nextest", "cargo-llvm-cov"],
+        )
```

Comment:
Not sure what's going on here, but `cargo install`ing in CI is a no go, and installing `rustup` doubly so. CI tries very hard to be as deterministic as possible, and only takes a dependency on Docker Hub and crates.io, with versions tightly pinned. We need to figure out how to get `cargo-llvm-cov` installed in the ci-builder image, and configured so it does not need to install any runtime dependencies.

Reply:
My mistake was choosing a super old version of cargo-llvm-cov because that was the first result on google. All good now.